--- a/munin-node.in	2019-09-15 18:22:00.832775862 -0500
+++ b/munin-node.in	2019-09-15 18:19:14.746122593 -0500
@@ -72,7 +72,7 @@
 for PLUG in $PLUGINS
 do 
   if [ "$PLUG" = "if_" ]; then  
-    for INTER in $(grep -E '^ *(ppp|eth|wlan|ath|ra|ipsec|tap|br-)[^:]{1,}:' /proc/net/dev | cut -f1 -d: | sed 's/ //g');
+    for INTER in $(grep -E '^ *(ppp|eth|wlan|ath|ra|ipsec|tap|br-|tun)[^:]{1,}:' /proc/net/dev | cut -f1 -d: | sed 's/ //g');
     do
       INTERRES=$(echo $INTER | sed -e 's/\./VLAN/' -e 's/\-/_/')
       RES="$RES if_$INTERRES"
@@ -80,7 +80,7 @@
       eval "config_if_${INTERRES}() { config_if $INTER $@; };"
     done
   elif [ "$PLUG" = "if_err_" ]; then
-    for INTER in $(grep -E '^ *(ppp|eth|wlan|ath|ra|ipsec|tap|br-)[^:]{1,}:' /proc/net/dev | cut -f1 -d: | sed 's/ //g');
+    for INTER in $(grep -E '^ *(ppp|eth|wlan|ath|ra|ipsec|tap|br-|tun)[^:]{1,}:' /proc/net/dev | cut -f1 -d: | sed 's/ //g');
     do
       INTERRES=$(echo $INTER | sed -e 's/\./VLAN/' -e 's/\-/_/')
       RES="$RES if_err_$INTERRES"
